src/plugins/lineendings/README.md

```diff
@@ -0,0 +1,13 @@
+- infos = Information about the lineendings plugin is in keys below
+- infos/author = Thomas Waser <>
+- infos/licence = BSD
+- infos/needs =
+- infos/provides = 
+- infos/placements = pregetstorage presetstorage precommit 
+- infos/description =
+
+## Introduction ##
+
+The Lineendings Plugin verifies the Lineendings of a file.
+If inconsistent lineendings or lineendings that don't match `config/valid` are detected the plugin yields an error.
```

Comment:
again, no `config/`

Can you explain `valid` a bit more? Does it contain a sequence of lineendings? If `\r\r\f` is given, how does it know if `\r\f` should be accepted?

Can you give an example how to use?